runexample

```diff
@@ -0,0 +1,79 @@
+#!/bin/bash
```

Comment:
The script content is not directly related with pipeline.

It also contains other thing like SOL, wordcount.....